lint_lib/parser.py

```diff
@@ -0,0 +1,131 @@
+from __future__ import unicode_literals
+
+import re
+
+from ._vendor.funcparserlib.lexer import Token, LexerError
+from ._vendor.funcparserlib.parser import (Parser, State, NoParseError,
+                                           finished, many, pure, skip, some)
+
+text_type = type("")
+binary_type = type(b"")
+
+
+def _make_tokenizer(specs):
+    # Forked from upstream funcparserlib.lexer to fix #44 and #46
```

Comment:
c.f. https://github.com/vlasovskikh/funcparserlib/issues/44 (won't fix) and https://github.com/vlasovskikh/funcparserlib/issues/46 (still open)